docs/changes.rst

```diff
@@ -10,6 +10,7 @@ To be released.
 - Fixed :exc:`KeyError` from :meth:`Entity.getlist()
   <wikidata.entity.Entity.getlist>` if the property is explictly associated
   with "no value". [:issue:`18`]
+- Better handling of non-existent entities
```

Comment:
Could you elaborate this?  Like, “what method became to return what when something does/is something,” blah blah.